tap_sftp/sync.py

```diff
@@ -44,6 +46,10 @@ def sync_stream(config, state, stream):
 
     return records_streamed
 
+@backoff.on_exception(backoff.expo,
```

Comment:
@harshpatel4crest add comments for this backoff

Reply:
Added comment.